lib/app/core/data/model/user_list_model.dart

```diff
@@ -0,0 +1,31 @@
+class UserList {
+  UserList({
```

Comment:
As this class is only responsible for providing one User object.
Instead of UserList, this can be renamed as UserData or UserModel.

Reply:
done